packages/adapter-vercel/index.js

```diff
@@ -164,19 +182,29 @@ export default function ({ external = [], edge, split } = {}) {
 
 			builder.log.minor('Writing routes...');
 
-			/** @type {Record<string, { path: string }>} */
-			const overrides = {};
-			builder.prerendered.pages.forEach((page, src) => {
-				overrides[page.file] = { path: src.slice(1) };
-			});
+			console.log(
+				JSON.stringify(
+					{
+						version: 3,
+						routes,
+						overrides
+					},
+					null,
+					'  '
+				)
+			);
```

Comment:
Is it intended to also be logging this to stdout, or was this just for debugging? Will that make things too noisy?

Reply:
it is not intended. oops